src/test/java/de/tum/in/www1/artemis/GuidedTourSettingResourceTest.java

```diff
@@ -39,17 +39,7 @@ public void tearDown() {
         database.resetDatabase();
     }
 
-    @Test
-    @WithMockUser(value = "student1")
-    public void guidedTourSettingsIsInitiallyNull() throws Exception {
-        User user = request.get("/api/account", HttpStatus.OK, User.class);
-        assertThat(user.getGuidedTourSettings().isEmpty()).isTrue();
-    }
-
-    @Test
-    @WithMockUser(value = "student1")
-    @SuppressWarnings("unchecked")
-    public void updateGuidedTourSettings() throws Exception {
+    public Set<GuidedTourSetting> createMockSettings() {
```

Comment:
why do you call this mock settings? isn't this just simple test data?

Reply:
Agreed, I will change the naming of the method :)